qa/tests/features/new/manual/filter-refiner/Range-Filter/range-filter.feature

```diff
@@ -69,7 +69,7 @@ Examples:
 Scenario Outline: 04 Empty "<Maximum Value>" field
 	When user clicks the "<Attribute Name>"
 	And enters the minimum value of range "<Minimum Value>"
-	And leaves the maximum value field empty
+	And maximum value field stays empty
 	And clicks the "Apply condition" button to apply the filter
 	Then the "<Attribute Name>" greater than "<Minimum Value>" should be displayed in the right part of the screen
 	And the "<Maximum Value>" should be counted as the maximum value
```

Comment:
It's not displayed anywhere how is the maximum value counted. What should user do? Check the number with filled maximum and compare? I think better is to check the number of variants
Here an below